lib/jekyll-feed.rb

```diff
@@ -12,6 +12,10 @@ def config
       @context.registers[:site].config
     end
 
+    def path
+      config["feed_path"] || "feed.xml"
```

Comment:
Is this actually the case? This `path` is only used as a string in the `link` tag rendered into the pages, it doesn't read the file at all.